splunk-apm/README.md

```diff
@@ -26,28 +28,37 @@ All wrapper layers are preconfigured to use Splunk as the tracing backend.
     ```
     OTEL_TRACES_EXPORTER: jaeger-thrift-splunk
     OTEL_EXPORTER_JAEGER_ENDPOINT: <Endpoint URL>
-    SPLUNK_ACCESS_TOKEN: orgAccessToken
+    SPLUNK_ACCESS_TOKEN: <Access token>
     ``` 
     By default, `OTEL_EXPORTER_JAEGER_ENDPOINT` is set to `http://127.0.0.1:9080/v1/trace` (local SmartAgent endpoint)
    
     Default span flush wait timeout is 1 second. It is controlled with a following environment variable (value in milliseconds):
     ```
     OTEL_INSTRUMENTATION_AWS_LAMBDA_FLUSH_TIMEOUT: 30000
     ```
-3. OpenTelemetry Metrics export
-
-    By default, OTEL metrics are disabled.    
     
-4. Sampling
+3. Sampling
 
     `OTEL_TRACE_SAMPLER` is set to `always_on` by default, meaning that all spans are always sampled.
 
-5. Logging
+4. Logging
     
     Logging is controller with `OTEL_LAMBDA_LOG_LEVEL` environment variable. Default set to `WARN`. When set to `DEBUG`, the logging exporter is added to traces exporter in order to help verify exported data.
     
-    > Enabling `DEBUG` generates additional logs, which may in turn increase AWS CloudWatch costs. 
+    > Enabling `DEBUG` generates additional logs, which may in turn increase AWS CloudWatch costs.
+
+5. OpenTelemetry Metrics export
+    
+    By default, OTEL metrics are disabled.
+
+6. SignalFx Metrics export
 
+   By default, SignalFx metrics are enabled with Splunk Metrics extension. Following configuration environment variables are required:
```

Comment:
```suggestion
   By default, metrics are enabled with the Splunk Metrics extension. Following configuration environment variables are required:
```

Reply:
Is there anything like "Splunk metrics" ? Or in other words - did we rebranded SFx to Splunk in all possible contexts? :)

Reply:
Made the suggestion brand agnostic. To be honest, I'm not 100% sure we've rebranded everything, but I'm pretty sure we will.